src/hooks/useMediaQuery/useMediaQuery.ts

```diff
@@ -1,9 +1,9 @@
 import React from "react";
 
-export default function useMediaQuery(queryInput: string): boolean {
+export default function useMediaQuery(queryInput: string): boolean | undefined {
   const query = queryInput.replace(/^@media( ?)/m, "");
 
-  const [match, setMatch] = React.useState(() => false);
+  const [match, setMatch] = React.useState<boolean | undefined>(undefined);
```

Comment:
comment (non-blocking): I wonder if `null` is more appropriate here. Happy to defer to the other reviewer I'm leaning towards what you have being fine but want to canvas opinion etc

Reply:
@edleeks87 that was my first idea to be honest. As `matchMedia` will eventually return boolean value `undefined` seemed to be a best "placeholder" in this context. Also it is idiomatic for TS and React for "not set yet", which exactly what is happening here.